app/uk/gov/hmrc/gform/auth/AuthModule.scala

```diff
@@ -36,11 +38,22 @@ class AuthModule @Inject() (configModule: ConfigModule, wSHttpModule: WSHttpModu
     override val http: HttpGet = wSHttpModule.auditableWSHttp
   }
 
-  val authActions: auth.Actions = new uk.gov.hmrc.play.frontend.auth.Actions {
+  lazy val authActions: auth.Actions = new uk.gov.hmrc.play.frontend.auth.Actions {
     def authConnector: AuthConnector = self.authConnector
   }
 
-  val authenticatedBy: auth.Actions#AuthenticatedBy = new authActions.AuthenticatedBy(governmentGateway, taxRegime, alwaysVisiblePageVisibility)
+  lazy val authorisationService: AuthorisationService = new AuthorisationService(eeittAuthorisationDelegate, authConnector)
+
+  lazy val authenticatedBy: auth.Actions#AuthenticatedBy = new authActions.AuthenticatedBy(governmentGateway, taxRegime, alwaysVisiblePageVisibility)
+
+  lazy val eeittConnector = new EeittConnector(
+    s"${configModule.serviceConfig.baseUrl("eeitt")}/eeitt",
+    wSHttpModule.auditableWSHttp
+  )
+
+  def redirectToEeitt(formTypeId: FormTypeId): Future[Result] =
```

Comment:
Module suppose to have only instances not functions. Lets move it to proper controller.